products/jbrowse-cli/src/commands/text-index.ts

```diff
@@ -36,7 +37,6 @@ export default class TextIndex extends JBrowseCommand {
 
     attributes: flags.string({
       description: 'Comma separated list of attributes to index',
-      default: 'Name,ID',
```

Comment:
I removed this default again in the attributes and exclude tag due to the way it was behaving with the meta and indexing. The order should be --attributes > config > defaults. However, since there was a default here, no matter what it would only pick up those attributes when getting information for the meta, making the results inaccurate. In addition, on line 201 I also changed that since the config would always be used even if the --attributes tag was used.

Reply:
IMO the order of operations should be config > --attributes > defaults

This to me is intuitive: if someone has gone through the work of specifying something individually on a track, it has the highest level of specificity, so it overrides --attributes

Reply:
I believe our original idea for the --attributes flag was to overwrite the config priority. That would be the case that someone does an aggregate index and wants all of them to be indexed with the same attributes. For some reason, I can't find the discussion when we initially were going to add --attributes though. By default the config always has priority UNLESS --attributes is specified

Reply:
I can see both sides to the debate, but to me it's a little like a css selector

I can globally apply a style to the document color:red but if a specific element has color:blue, it gets to be color:blue